scss/_button-group.scss

```diff
@@ -76,12 +76,12 @@
   padding-left: $btn-padding-x * .75;
 
   &::after,
-  .dropup &::after,
-  .dropright &::after {
+  .drop-bs &::after,
+  .drop-ie &::after {
```

Comment:
These abbreviations are a little cryptic—I took `bs` and `ie` as Bootstrap and IE. 😅 Haven't reviewed everything here, still thinking on the rest.

Reply:
Indeed—it'll be dropped for now, but we probably have to get used to it in a (not so) far future since `*-block-start` and `*-inline-end` are the new `*-top` and `*-right` :D